code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -747,6 +747,9 @@ var/list/ai_list = list()
 	apc.malfvacate()
 
 /mob/living/silicon/ai/proc/toggle_camera_light()
```

Comment:
Here's an issue, your pushing a proc into the AI commands tab, which is not a good. Make a verb to call the proc.